.github/workflows/connector_ops_ci.yml

```diff
@@ -21,3 +22,35 @@ jobs:
         run: pip install --quiet -e ./tools/ci_connector_ops
       - name: Check test strictness level
         run: check-test-strictness-level
+  requires-connector-team-review:
+    name: "Check if a review is required from Connector teams"
+    runs-on: ubuntu-latest
+
+    # GitHub should provide a "pull_request_review_target", but they don't and
+    # the action will fail if run on a forked PR.
+    if: github.event.pull_request.head.repo.full_name == github.event.pull_request.base.repo.full_name
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Install Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.9"
+      - name: Install ci-connector-ops package
+        run: pip install --quiet -e ./tools/ci_connector_ops
+      - name: Check if the PR requires a connector team review
+        id: check-requires-connector-team-review
+        run: check-requires-connector-team-review >> $GITHUB_OUTPUT
+      - name: Check if the required teams reviewed
+        if: steps.check-requires-connector-team-review.outputs.REQUIRES_CONNECTOR_TEAM_REVIEW == 'True'
```

Comment:
[not-blocking] The way python stringifies "true" is so strange.  Maybe the output should be "true" or "1"?

Reply:
👌 I'll make it return `true` or `false`